### ISSUE TITLE
error when using any embedded aesthestic in aesthetic.py

### ISSUE THREAD

**Issue by TimoPaschke** [AUTOMATIC1111/stable-diffusion-webui-aesthetic-gradients#10]
Hello, 
I installed aesthetic gradiants using the SD webui (https://github.com/AUTOMATIC1111/stable-diffusion-webui-aesthetic-gradients).

My fresh installation of SD with model v1.5 works just fine until I try to make use of any pretained aesthetics. 
Attached you find a screenshot of my SD instance running up, creating one image with Euler (30 steps) and then I tried to use the "sac_8plus" aesthetic.
<img width="1239" alt="aesthetics_error" src="https://user-images.githubusercontent.com/391933/199435023-1f296127-fcad-48d8-aaa0-64dbd16782fe.png">

What did I do wrong? Is there a workaround for this?

Best regards,
Theo

PS: This is the list of extensions I installed only by using the web-installer of SD:
<img width="567" alt="aesthetics_error_extensions" src="https://user-images.githubusercontent.com/391933/199435275-b5ccd22e-290e-4e1c-a8a1-3c2b9db4f353.png">

**Comment by thenolifer** [comment]
hey, I'm getting exactly the same error with those pre-trained aesthetics. They worked previous to installing this extension via the WebUI interface. 

Interestingly, I trained one and it runs fine - just the pre-made ones, glowwave, gloomcore, cloudcore, etc, have stopped working.
